### DISCUSSION TITLE
Display audio thumbnail on it's playing page

### DISCUSSION THREAD

**Discussion by People-11** [spencerwooo/onedrive-vercel-index#389] [Ideas]
Now this project can render media thumbnail in the Grid layout.
![Snipaste_2022-02-09_18-26-48](https://user-images.githubusercontent.com/41558215/153180107-19598a7d-d9fd-40a6-a483-3d6b735be547.PNG)

How about rendering audio thumbnail on it's playing page?
![image](https://user-images.githubusercontent.com/41558215/153179566-41d227a7-66e0-4ca3-bcda-cab035a41d6d.png)

Show the music icon if the audio doesn't contain thumbnail.

**Comment by spencerwooo** [comment]
Can you link a music file with an available thumbnail for me to test with? Thx.

**Reply by People-11** [reply]
> Can you link a music file with an available thumbnail for me to test with? Thx.

https://people11.vercel.app/音乐/星の消えた夜に%20-%20Aimer/10.%20my%20sweetest%20one%20(Remaster%20for%20星の消えた夜に%202022).flac

**Reply by Android-KitKat** [reply]
> > Can you link a music file with an available thumbnail for me to test with? Thx.
> 
> [https://people11.vercel.app/音乐/星の消えた夜に%20-%20Aimer/10.%20my%20sweetest%20one%20(Remaster%20for%20星の消えた夜に%202022).flac](https://people11.vercel.app/%E9%9F%B3%E4%B9%90/%E6%98%9F%E3%81%AE%E6%B6%88%E3%81%88%E3%81%9F%E5%A4%9C%E3%81%AB%20-%20Aimer/10.%20my%20sweetest%20one%20(Remaster%20for%20%E6%98%9F%E3%81%AE%E6%B6%88%E3%81%88%E3%81%9F%E5%A4%9C%E3%81%AB%202022).flac)

好耶，大佬网盘有瑞克和莫蒂第五季。(　ﾟ∀ﾟ) ﾉ♡
我之前找不到资源。

**Reply by People-11** [reply]
是的，这是我的公开网盘，正如此项目的特性一样，里面的东西都可以在线预览或免费下载。
我比较擅长寻找与整理网上的资源，所以当我/我的朋友有什么需要时（电影，音乐，游戏或者其他资源），我会搜索,整理并上传到这里面。
我称它为小纸箱，也欢迎你加收藏，有时间就在里面翻翻，也许恰好有你想要的东西。
你也可以向我寻求资源（当然不要在这里说以免off topic），我有时间就会寻找。

**Comment by spencerwooo** [comment]
https://drive.spencerwoo.com/%F0%9F%A5%9F%20Some%20test%20files/Videos%20and%20music/my%20sweetest%20one%20(Remaster%20for%20%E6%98%9F%E3%81%AE%E6%B6%88%E3%81%88%E3%81%9F%E5%A4%9C%E3%81%AB%202022).flac

![CleanShot 2022-02-10 at 13 45 39](https://user-images.githubusercontent.com/32114380/153345249-cd0cef43-7c4b-4804-874c-db01a4509979.png)

**Reply by People-11** [reply]
I deleted spin, rounded full and m-4 so it looks like this
![image](https://user-images.githubusercontent.com/41558215/153378703-88f716e7-f480-4f34-8a8f-7d9de2125f11.png)

**Comment by magisk317** [comment]
![image](https://user-images.githubusercontent.com/93979778/210179198-7da1e21e-29f6-4113-a3c5-bb4f75422792.png)

抱歉打扰，我的跟你一样的flac格式的音乐，但是无法播放，时长显示为0，也没有封面，本地播放却没有问题，想问下有可能是哪里的问题呢？（MP3格式没问题，可以正常播放）
![image](https://user-images.githubusercontent.com/93979778/210179234-acaa4d8f-b30f-4dd8-8610-6276d90752c0.png)
